corehq/apps/hqadmin/reports.py

```diff
@@ -495,3 +498,64 @@ def headers(self):
     @property
     def rows(self):
         return self.table_data.rows
+
+
+class StaleCasesTable:
+
+    STALE_DATE_THRESHOLD_DAYS = 365
+
+    @property
+    def headers(self):
+        return DataTablesHeader(
+            DataTablesColumn(gettext_lazy("Domain")),
+            DataTablesColumn(gettext_lazy("Case count"))
+        )
+
+    @property
+    def rows(self):
+        rows = []
+        case_count_by_domain = self._stale_case_count()
+        for bucket in case_count_by_domain.values():
+            rows.append([bucket.key, bucket.doc_count])
+        return rows
+
+    def _stale_case_count(self):
+        return (
+            CaseSearchES()
+            .is_closed(False)
+            .server_modified_range(lt=self._get_stale_date())
+            .size(0)
+            .aggregation(
+                TermsAggregation('domain', 'domain.exact')
+            )
+        ).run().aggregations.domain.buckets_dict
+
+    def _get_stale_date(self):
+        current_date = datetime.now()
+        stale_threshold_date = current_date - timedelta(days=self.STALE_DATE_THRESHOLD_DAYS)
+        return stale_threshold_date
+
+    @staticmethod
+    def format_as_table(row_data, data_tables_header):
```

Comment:
This is a useful function, but it is more generic than this class, and it expects its headers to be a list of DataTablesHeader instances. For those reasons, I would pull it out of here, move it somewhere under `corehq/apps/reports/datatables/`, and maybe rename it. If you feel that having a method is useful, you could change this to something like:
```python
    def format_as_table(self):
        return datatable_as_text(self.rows, self.headers)
```
(Obviously assumes that `self.rows` doesn't run the query again.)

(Not applicable here because it's probably not worth an additional requirement, but useful to know about: [Tablib](https://tablib.readthedocs.io/en/stable/) is cool, and [it can do this](https://tablib.readthedocs.io/en/stable/formats.html#rst). (Also, I'm biased, because I wrote that bit.))

Reply:
That is a good point. We no longer need this function as I'll be refactoring the task to save the data to a csv file instead. I'm therefore going to completely remove this method. 
I thought it might be useful to keep it around and put it in `corehq/apps/reports/datatables` as you suggested for future use but, in saying that, YAGNI.